src/docs/concepts-api-sdk/rate-limiting/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Rate limiting in Marketplace API and Integration API
+slug: rate-limiting
+updated: 2022-12-12
+category: concepts-api-sdk
+ingress:
+  This article describes rate limiting in Marketplace API and
+  Integration API, as well as suggests how to handle those limits
+  gracefully.
+published: true
+---
+
+Starting in XX-XX-2023, Marketplace API and Integration API will feature
+new rate limits in dev and demo environments. In addition, Integration
+API will feature new concurrency limits in dev and demo environments
+and, eventually, in production environments.
+
+<extrainfo title="What are rate limits and concurrency limits?">
+
+**Rate limit** means a limit on the number of requests from your
+marketplace that the API can process within a time span, e.g. in one
```

Comment:
"From your marketplace" seems a bit too vague, I think. Our definition for the rate limit is that it applies on per client IP manner, i.e. each unique IP is rate limited separately. However, perhaps since this is more of a general definition of the terms, perhaps it's ok to skip that bit? How about just "limit on the number of requests to the API within a time span..."?

Reply:
Omitted "from your marketplace" in both contexts, since it's indeed a general explanation rather than a full definition.